src/controllers/project.controller.ts

```diff
@@ -119,13 +121,68 @@ export class ProjectController {
 
 		this.view.update({ type: MessageType.setSessionsSection, payload: sessionsViewObject });
 
+		const activeDeployment = deployments?.find((deployment) => deployment.state === DeploymentState.ACTIVE_DEPLOYMENT);
+
 		if (this.selectedDeploymentId) {
 			await this.selectDeployment(this.selectedDeploymentId);
 		}
+
+		if (!activeDeployment) {
+			return;
+		}
+		const { data: buildDescription, error: buildDescriptionError } = await BuildsService.getBuildDescription(
+			activeDeployment.buildId
+		);
+
+		if (buildDescriptionError) {
+			commands.executeCommand(
+				vsCommands.showErrorMessage,
+				`${translate().t("errors.buildInfoFetchFailedForBuild", {
+					buildId: activeDeployment?.buildId,
+					error: (buildDescriptionError as Error).message,
```

Comment:
The notification should be - "Start session failed"
In the log should be the full data/description